common/djangoapps/entitlements/management/commands/expire_old_entitlements.py

```diff
@@ -0,0 +1,76 @@
+"""
+Management command for expiring old entitlements.
+"""
+
+import logging
+
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+
+from entitlements.models import CourseEntitlement
+from entitlements.tasks.v1.tasks import expire_old_entitlements
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+PAGE_SIZE = 10000  # arbitrary, should be adjusted if we see performance issues in the wild
```

Comment:
Should this be something that can be configured rather than hard coded?

Reply:
Could also just be a command line argument maybe?

Reply:
I'd push back against configuration (I don't love configuration bloat, especially if we don't know when/if we'd want to change this yet).

But a command line argument with a default sounds great.  Easier to adjust once this is out if my very scientifically chosen page size is bad.

Reply:
Done via a new --batch-size argument.